### ISSUE TITLE
Expand Smart Filter - Search Custom Content Types

### ISSUE THREAD

**Issue by LoganT8** [CrocoBlock/suggestions#7644]
Hello,

We recently purchased Crocoblock to test the LaborTime (Job Board) template.  This looked extremely promising and we fell in love with how easy it was to profiles on the front end.  However we found an issue with the inability to search resumes/jobs that is having us reconsider if Crocoblock is the right solution.

In order for a job board website to function correctly, the search function needs to match results from Custom Content Types: Work Experience, Training/Certifications, Education.  I made a flow chart with the User Profile and one CCT being pulled together into a Smart Search. 

![image](https://github.com/user-attachments/assets/c4254de5-c08d-42c7-a01c-08bc00c117ab)

**Comment by Crocoblock** [comment]
@LoganT8  Hi!

Please clarify your request, would you like to use JetSearch to search for CCT products?

**Comment by LoganT8** [comment]
Hi @Crocoblock,

We would like to take input fields from Custom Content Types (work experience) and include them in a JetSearch with the User fields.

The easiest way to understand what we are trying to achieve is to go to the Crocoblock demo site and Search Resumes for the word Web Designer (https://demo.crocoblock.com/jobboard/find-resumes/).  

You will get zero results, but if you look at the profiles you will see they have experience as a web designer
https://demo.crocoblock.com/jobboard/profile/anfisa/profile-employee/
https://demo.crocoblock.com/jobboard/profile/born/profile-employee/

Ideally we would be want to search users for _employee-name,_experience-job-title,_experience-company

**Comment by LoganT8** [comment]
Any update on this?
![Screenshot 2024-08-08 140851](https://github.com/user-attachments/assets/a82a9bbb-5ad7-4bf7-b87f-87cac290e6c3)

**Comment by curiousprojects** [comment]
Yes, I would like Jetsearch to be able to search CCT's - I have 10,000 real estate listings and no way to offer a main page search, only using Jetfilters.

**Comment by LoganT8** [comment]
@curiousprojects reach out to me sometime (logantate48@gmail.com).  I build large directory sites as well - might have a solution for you!
